ygot/diff.go

```diff
@@ -446,6 +446,19 @@ type DiffPathOpt struct {
 	// -ignore_shadow_schema_paths flag, and therefore have the
 	// "shadow-path" tag.
 	PreferShadowPath bool
+	// Explicitly delete and then update leaf-list, which is different and exists in both orig and modified.
+	// For example, if the original path is x/y/[a, b] and the modified leaf-list is x/y/[b, c],
+	// Then resulting notification will contain:
+	//   - Delete { Path: x/y }
+	//   - Update { Path: x/y Value: [b,c] }
+	// If this is false, then the leaf-list will be updated directly to the new value
+	// without the delete step.
+	// This is required because if the leaf-list is updated directly,
+	// the resulting notification will be:
+	//   - Update { Path: x/y Value: [b,c] }
+	// This will cause the new value to be appended in the OC, but the original value
+	// will still be present in the OC. Resultant OC in the device will be x/y/[a, b, c].
```

Comment:
```suggestion
```

Suggest to remove these lines, since it's not clear that this is necessarily the right, or standard behaviour.